R/FilterState.R

```diff
@@ -90,16 +105,22 @@ InteractiveFilterState <- R6::R6Class( # nolint
       }
       private$dataname <- dataname
       private$varname <- varname
-      private$varlabel <- if (identical(varlabel, as.character(varname))) {
-        # to not display duplicated label
-        character(0)
-      } else {
-        varlabel
-      }
-      private$extract_type <- extract_type
       private$selected <- reactiveVal(NULL)
+      private$keep_na <- reactiveVal(keep_na)
+      private$keep_inf <- reactiveVal(keep_inf)
+      private$extras <- list(...)
+      private$fixed <- fixed
+      # Establish varlabel.
+      varlabel <- attr(x, "label")
```

Comment:
If there is no label this would be NULL I think. Would an empty string be more appropriate?